src/current/cockroachcloud/configure-scim-provisioning.md

```diff
@@ -9,40 +9,52 @@ docs_area: manage
 
 When a user is directly or indirectly unassigned, their {{ site.data.products.db }} account is disabled or removed, depending on the capabilities of your IdP. To remove a disabled user from {{ site.data.products.db }}, refer to [Manage an Organization's Members](managing-access.html#manage-an-organizations-users).
 
 {{site.data.alerts.callout_info}}
 Okta disables deprovisioned users and does not support deleting them.
 {{site.data.alerts.end}}
 
-To learn more about user and group assignments and group push on Okta, refer to the following topics in the Okta documentation:
+### SCIM group push
 
-- [Manage Users](https://help.okta.com/en-us/Content/Topics/users-groups-profiles/usgp-people.htm)
-- [Manage Groups](https://help.okta.com/en-us/Content/Topics/users-groups-profiles/usgp-groups-main.htm)
-- [Manage Group Push](https://help.okta.com/en-us/Content/Topics/users-groups-profiles/usgp-group-push-main.htm)
+User accounts are provisioned only based on assignments in the SCIM app integration. Assigning an IAM group to the app integration is equivalent to assigning each of the group's members individually. Assigning a group to the app integration does not create a group and links its members in {{ site.data.products.db }}, unless [Group Push](#automate-group-management) is enabled. When {{ site.data.products.db }} has no awareness of group membership, role assignments must be applied to each user account separately.
 
-Before following these instructions, if your IdP is Okta, then it may be helpful to read Okta's [article about SCIM](https://developer.okta.com/docs/concepts/scim/), as well as [Configure provisioning for an app integration
-](https://help.okta.com/en-us/Content/Topics/Provisioning/lcm/lcm-provision-application.htm) in the Okta documentation. Otherwise, refer to your IdP's documentation about configuring SCIM.
+Depending on what it supports, your IdP may provide a mechanism to sync IAM groups as groups in {{ site.data.products.db }}. In Okta, this functionality is called _Group Push_. When this functionality is configured:
+
+1. You can configure the SCIM app integration to selectively synchronize details about an IAM group to {{ site.data.products.db }}, including the group's name and membership list. In Okta, this group is said to be _pushed_.
+1. Within {{ site.data.products.db }}, you can assign roles to a pushed group, and those roles are automatically assigned to the individual group members with provisioned accounts in {{ site.data.products.db }}.
+1. An account is only ever automatically provisioned in {{ site.data.products.db }} based on assignments in the SCIM app integration.
+
+    - If a group is pushed but not assigned to the SCIM app in your IdP, roles can be granted to the group in {{ site.data.products.db }}, and group members who are already provisioned in {{ site.data.products.db }} or who are assigned to the app integration in the future, automatically receive those roles.
```

Comment:
Again, I think we should indicate that this is all applicable to Okta specifically, and we don't know the behavior with other IdPs yet.

Reply:
This whole section is about Okta, see line 42.